### PR TITLE
Add jupyter-kernel-client

### PR DESCRIPTION
**Pull request by fcollonval**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~~If static libraries are linked in, the license of the static library is packaged.~~
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Review by fcollonval — COMMENTED** [review]
@echarles could you confirm you accept to be added as maintainer of this recipe?

**Comment by echarles** [comment]
> @echarles could you confirm you accept to be added as maintainer of this recipe?

Sure, I agree

**Comment by fcollonval** [comment]
@conda-forge/help-python, ready for review!

File: recipes/jupyter-kernel-client/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/jupyter-kernel-client/meta.yaml:1]: Ping.

**Comment by fcollonval** [comment]
Thanks for the review @ocefpaf. I applied the suggestions.

**Pull request merged**